packages/design-system-docs/src/pages/components/ChoiceList/_ChoiceList.docs.scss

```diff
@@ -93,10 +93,14 @@ Style guide: components.choice.react
 
 The following Sass variables can be overridden to theme choice fields:
 
+- `$choice-size`
```

Comment:
I think it would be good to put these into a table with descriptions of what each variable does - USWDS follows a similar approach https://designsystem.digital.gov/components/alert/#component-settings 
![Screen Shot 2020-09-24 at 3 25 14 PM](https://user-images.githubusercontent.com/1449852/94190467-25b05600-fe7a-11ea-8dc6-9cfab30db4ea.png)

Reply:
yeah, we should make a ticket for that for all the components and utilities I think. It would also be nice to redesign the guidance section in the future and organize it better :/

Reply:
I'm going to add a ticket for it now 👍